odoo/models.py

```diff
@@ -3476,6 +3476,9 @@ def _setup_complete(self):
         # register constraints and onchange methods
         cls._init_constraints_onchanges()
 
+        if cls._check_company_auto is not None and 'company_id' not in cls._fields and self._name != 'res.company':
```

Comment:
Why make a difference between `False` and `None`?

Reply:
But just don't check it if it is `False` then, only when it is `True`

Reply:
Oh, right.

Reply:
No, no, my bad. I was to swiftly to answer. I check with `cls._check_company_auto is not None` as a way to detect if a model has declared `_check_company_auto`, whether if it's has been to True or to False, in this case.